instrumentation/log4j/log4j-2.13.2/library/src/test/resources/log4j2-test.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN" packages="com.example.appender">
```

Comment:
Do you want to demonstrate the usage of context data?

Reply:
Good idea